src/card-team-edit.js

```diff
@@ -54,132 +48,192 @@ export default class CardTeamEdit extends expose.Component {
+                        className={classes.publicLinkCopyButton}
+                        src="/server/assets/copy.svg"
+                        alt="copy"
+                      />
+                    </span>
+                  </div>
+                  <input
+                    id="publicLink"
+                    readOnly
+                    size={publicLink.length}
+                    value={publicLink}
+                    className={classes.publicLink}
+                  />
+                </div>
+              )}
+            </CardSection>
+          </>
+        )}
+        <CardSection>
+          <div
+            className="edit-button button confirm-button"
+            onClick={this.handleConfirmClick}
+          >
+            <img
+              className="edit-button-icon"
+              src="/server/assets/check.svg"
+              alt=""
+            />
+            <span className="edit-button-icon"> Save </span>
+          </div>
+          <div
+            className="edit-button button cancel-button"
+            onClick={this.handleCancelClick}
+          >
+            <img
+              className="edit-button-icon"
+              src="/server/assets/cancel.svg"
+              alt=""
+            />
+            <span className="edit-button-icon"> Cancel </span>
+          </div>
+          {!this.props.isNew && (
+            <div
+              className="edit-button button cancel-button"
+              onClick={this.handleDeleteClick}
+            >
+              <img
+                className="edit-button-icon"
+                src="/server/assets/delete.svg"
+                alt=""
+              />
+              <span className="edit-button-icon"> Delete </span>
+            </div>
+          )}
+        </CardSection>
+      </Card>
     );
   }
-};
+}
+
+const styles = css => ({
+  publicLink: {
+    fontSize: css.typography.size.xSmall,
+    padding: css.spacing.xxSmall,
+    backgroundColor: css.colors.INVISIBLE,
+    color: css.colors.TEXT_LIGHT,
+    border: '0px',
+    resize: 'none',
+    whiteSpace: 'unset',
+    overflowWrap: 'unset',
+  },
+  publicLinkLabelBox: {
+    fontSize: css.typography.size.large,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+  },
+  publicLinkLabel: {
+    paddingRight: css.spacing.xxSmall,
+  },
+  publicLinkLineItem: {
+    padding: css.spacing.xxSmall,
+  },
+  publicLinkCopyButton: {
+    width: css.sizes.ICON,
+    cursor: 'pointer',
+  },
+  publicLinkCopiedText: {
+    opacity: 0,
+  },
+  publicLinkContainer: {
+    backgroundColor: css.colors.PRIMARY_DARK,
+    borderRadius: css.spacing.xSmall,
+    display: 'flex',
+    justifyContent: 'flex-start',
+    alignItems: 'center',
+    marginTop: css.spacing.xxSmall,
+    overflow: 'hidden',
+  },
+  publicLinkCheckbox: {
+    width: '1rem',
+    height: '1rem',
+    marginRight: css.spacing.xSmall,
+  },
+});
+
+export default injectSheet(styles)(CardTeamEdit);
```

Comment:
This is probably how we should start doing css?  It's better than inlining the css in the components (more flexible) and we don't need to go to a separate file and "build-css" to get changes.  Luckily this works with the current setup, so no need to do a big pass on things.  Can do this incrementally.